avb/libavb/uefi_avb_ops.c

```diff
@@ -271,7 +279,11 @@ static AvbIOResult write_rollback_index(__attribute__((unused)) AvbOps* ops,
     ret = EFI_SUCCESS;
   else {
 #if defined(SECURE_STORAGE_EFIVAR)
-    ret = write_efi_rollback_index(rollback_index_slot, rollback_index);
+#ifdef USE_TPM
```

Comment:
I still don't like this entire file imbrication of `#ifdef` but I suppose that you don't want to clean that stuff now and I understand that.